### PR TITLE
[DOC] More on paths and lists

### PR DESCRIPTION
**Pull request by BurdetteLamar**
Improves consistency among method sthat have arguments `src` (path or array) and `dest` (path):  `cp`, `cp_lr`, `cp_r`, `install`, `ln`, `ln_s`, `ln_sf`, `mv`.

### CODE REVIEW

File: lib/fileutils.rb [hunk 1]
```diff
@@ -1018,14 +1018,15 @@ def copy_stream(src, dest)
...
-  # first copies, then removes +src+.
+  # Moves entries.
   #
-  # Arguments +src+ and +dest+
+  # Arguments +src+ (a single path or an array of paths)
+  # and +dest+ (a single path)
   # should be {interpretable as paths}[rdoc-ref:FileUtils@Path+Arguments].
   #
+  # If +src+ and +dest+ are on different devices,
+  # first copies, then removes +src+.
```
Comment (peterzhu2118) [inline lib/fileutils.rb:1028]: I think the definition of "device" is confusing and ambiguous. I think of one drive as a "device", but a drive can have multiple file systems.

The man page for `mv` says the following:

> As the rename(2) call does not work across file systems, mv uses cp(1) and rm(1)
> to accomplish the move.

```suggestion
  # If +src+ and +dest+ are on different file systems,
  # first copies, then removes +src+.
```

File: lib/filutils.sav.rb [hunk 2]
```diff
@@ -0,0 +1,2528 @@
+# frozen_string_literal: true
```
Comment (peterzhu2118) [inline lib/filutils.sav.rb:1]: I think this file was committed on accident?

File: lib/fileutils.rb (hunk 1 above)
Comment (BurdetteLamar) [reply lib/fileutils.rb]: Fixed (three places).

File: lib/filutils.sav.rb (hunk 2 above)
Comment (BurdetteLamar) [reply lib/filutils.sav.rb]: Removed.

**Review by peterzhu2118 — APPROVED** [review]
Looks good!

**Pull request merged**